### ISSUE TITLE
 Unable to find a specification for 'Mantle' while Pod Install

### ISSUE THREAD

**Issue by KiranJasvanee** [ChatSecure/ChatSecure-iOS#1142]
I've followed the instructions has been mentioned in ReadMe.
While 'pod install' I'm constantly getting below error.
`Unable to find a specification for 'Mantle'`
I've tried to remove my pod repo and setup again. still, the issue persists.

These are my logs 
```
$ pod install
Analyzing dependencies
Fetching podspec for `CPAProxy` from `Submodules/CPAProxy/CPAProxy.podspec`
Fetching podspec for `ChatSecure-Push-iOS` from `Submodules/ChatSecure-Push-iOS/ChatSecure-Push-iOS.podspec`
Fetching podspec for `ChatSecureCore` from `ChatSecureCore.podspec`
Fetching podspec for `IOCipher` from `Submodules/IOCipher/IOCipher.podspec`
Fetching podspec for `JSQMessagesViewController` from `Submodules/JSQMessagesViewController/JSQMessagesViewController.podspec`
Fetching podspec for `LumberjackConsole` from `Submodules/LumberjackConsole/LumberjackConsole.podspec`
Pre-downloading: `Mantle` from `https://github.com/ChatSecure/Mantle.git`, commit `4c1a09cb0c0811956cd35262340e42b940971cbb`
[!] Unable to find a specification for 'Mantle'.
Kirans-MacBook-Pro:ChatSecure-iOS smartsense-kiran$ 
```

**Comment by afriedmanGlacier** [comment]
We are having the same issue

**Comment by chrisballinger** [comment]
The spec for Mantle is located in the Podspecs directory for the master branch: 

```ruby
pod 'Mantle', :podspec => 'Podspecs/Mantle.podspec.json'
```

Perhaps you're using an older commit?

**Comment by afriedmanGlacier** [comment]
Mine has Mantle.podspec.json in the Podspecs directory and is pulling the same commit that @KiranJasvanee shows: 4c1a09cb0c0811956cd35262340e42b940971cbb

Mantle.podspec.json just points to the 2.1.0_headerfix branch. And I see that branch has one more commit since 4c1a09cb0c0811956cd35262340e42b940971cbb that includes the podspec file in the Mantle project, so I'm not sure why it's pulling that commit rather than the one you submitted in June @chrisballinger

**Comment by chrisballinger** [comment]
Maybe your Podfile.lock needs to be updated. Can you try running `bundle exec pod update Mantle`

**Comment by afriedmanGlacier** [comment]
I tried that. Result was:

Could not find CFPropertyList-3.0.0 in any of the sources
Run `bundle install` to install missing gems.

So I ran "bundle install" and then re-ran "bundle exec pod update Mantle" and it failed on installing OTRKit. I tried a few more things and then re-ran it, and the last time it succeeded.

**Comment by chrisballinger** [comment]
Nice! I think I know why this issue wasn't caught by CI

**Comment by chrisballinger** [comment]
Fixed in a2a33a53
